assets/stylesheets/desktop/chat-message.scss

```diff
@@ -0,0 +1,17 @@
+.chat-msgactions {
+  .react-btn,
+  .reply-btn,
+  .bookmark-btn {
+    border: 1px solid transparent;
+    border-bottom-color: var(--primary-low);
+    border-radius: 0;
+    border-top-color: var(--primary-low);
+
+    &:hover {
+      background: var(--primary-low);
+      border-color: var(--primary-low-mid);
+      color: var(--primary-medium);
+      z-index: 1;
```

Comment:
?

Reply:
I think Charlie added this?

Reply:
Yes I did. Well, I moved it. Why? Did I/it break something?

Reply:
Just wondering why we need this?

Reply:
This styling was in common, and since there's now a different styling for mobile/desktop because the borders & bg are removed on mobile, I just figure to move this styling to desktop specific and included the hover too for good measure.

Reply:
talking of the `z-index: 1` why do we need this?

Reply:
That I didn't add. Like iI said, I merely moved the code. I don't feel comfortable touching anything I don't have to in case it explodes 😜

Reply:
ok